cardano-db-sync/src/Cardano/DbSync/Api/Types.hs

```diff
@@ -78,6 +78,7 @@ data InsertOptions = InsertOptions
   , ioRewards :: !Bool
   , ioMultiAssets :: !Bool
   , ioMetadata :: !Bool
+  , ioKeepMetadataNames :: ![Text]
```

Comment:
I think `![a]` is a bit of a misnomer, because (if I understand correctly) it will only evaluate the head of the list